docgen/src/main/java/com/google/errorprone/BugPatternFileGenerator.java

```diff
@@ -114,18 +115,21 @@ private static MessageFormat constructPageTemplate(
         + " docs/bugpattern.\n"
         + "-->\n\n");
 
-    result.append("<div style=\"float:right;\"><table id=\"metadata\">\n"
-        + "<tr><td>Category</td><td>{3}</td></tr>\n"
-        + "<tr><td>Severity</td><td>{4}</td></tr>\n"
-        + "<tr><td>Maturity</td><td>{5}</td></tr>\n"
-        + "</table></div>\n\n"
-        + "# Bug pattern: {1}\n"
-        + "__{8}__\n");
+    if (!generateFrontMatter) {
```

Comment:
I wanted the AUTO-GENERATED comment as early as possible, but it can't be before the frontmatter.